x-pack/plugins/reporting/server/export_types/csv_from_savedobject/server/lib/generate_csv_search.ts

```diff
@@ -4,6 +4,7 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { ReportingCore } from '../../../../';
```

Comment:
Note: I'm using the "Organize Imports" feature in VS Code to straighten these out. Not sure why it put this one at the top, but it's ok I guess.